frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -359,6 +360,11 @@ const Profile: NextPage = () => {
 
   const banners = (
     <section className={styles["banners-wrapper"]}>
+      {!profile.has_premium &&
+      isPeriodicalPremiumAvailableInCountry(runtimeData.data) &&
+      isFlagActive(runtimeData.data, "premium_promo_banners") ? (
```

Comment:
@maxxcrawford I moved the conditions to `profile.page.tsx`, it seems to make more sense to keep track of the logic of where and when it should be displayed.